Makefile

```diff
@@ -111,19 +111,10 @@ install-yamllint:
 yamllint:
 	yamllint .
 
-# Check semantic convention policies on YAML files
-.PHONY: check-policies
-check-policies:
-	docker run --rm -v $(PWD)/model:/source -v $(PWD)/policies:/policies -v $(PWD)/templates:/templates \
-		otel/weaver:${WEAVER_VERSION} registry check \
-		--registry=/source \
-		--diagnostic-format=ansi \
-		--policy=/policies/registry.rego
-
 # Test rego policies
 .PHONY: test-policies
 test-policies:
-	docker run --rm -v $(PWD)/policies:/policies openpolicyagent/opa:0.67.1 test --explain fails /policies
+	docker run --rm -v $(PWD)/policies:/policies openpolicyagent/opa:0.67.1 test --var-values --explain fails /policies
```

Comment:
I don't see `--var-values` in your PR (it prints values of vars which is useful in my parametrized tests), but let's merge your PR first so that I can check this one against your fixes in makefile.

Reply:
OH - i meant I fixed the issue that caused failures in tests. --var-values should be added, I must not have submitted that Makefile in that PR, let's make sure it's added.